src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/add-lets-encrypt-txt-record.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: How to add a TXT record to configure Let's Encrypt certificate
+description: After you create a Let’s Encrypt certificate in a platform other than Azion, but want to use it with a zone created in Intelligent DNS, you must also create TXT record to add the information.
+meta_tags: secure, intelligent dns, record, certificate, domain, let's encrypt
+namespace: docs_guides_secure_intelligent_dns_lets_encrypt_record
+permalink: /documentation/products/guides/secure/lets-encrypt-record/
+menu_namespace: secureMenu
+---
+
+import Button from '~/components/Button.astro'
+
+:::caution[warning]
+This guide covers a scenario in which you have a Let's Encrypt™ certificate created in a platform *outside* Azion and want to associate it with an Intelligent DNS zone. If you've created a Let's Encrypt certificate directly on Azion, the process is automatic and you don't need to create a TXT record. Find out more in the [Let's Encrypt guide](/en/documentation/products/guides/how-to-generate-a-lets-encrypt-certificate/).
+:::
+
+After you create a Let’s Encrypt certificate in a platform other than Azion, but want to use it with a [zone created in Intelligent DNS](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you must also create **TXT record** in Intelligent DNS to add the information given by the Let’s Encrypt provider. This helps verify the domain's DNS is owned by those who requested the certificate.
```

Comment:
Sugiro mudar toda linha 16 para:

While requesting the Let's Encrypt certificate over [DNS challenge](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) method, you'll have to add a specific TXT record to your DNS zone to ensure the certificate validation. Assuming that you're hosted it on [Intelligent DNS](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), follow the instructions below:

Reply:
MOTIVO: "After you create" pressupõe que o certificado já foi criado, quando na verdade, essa etapa do TXT ocorre durante o processo de solicitação dele.